recipes/scsplit/meta.yaml

```diff
@@ -11,25 +11,33 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('scsplit', max_pin="x.x.x") }}
 
 requirements:
   host:
-    - python
-    - pip
+    - python =3.9
   run:
-    - python
+    - python =3.9
+    - numpy
+    - pandas <2.0
+    - pyVCF
+    - pysam
+    - scikit-learn ==1.1.3
+    - scipy
+    - setuptools <58
 
 test:
```

Comment:
it would be enough to just run `scSplit --version` or something similar as test. Please do not include test data into this repo.

Reply:
The package doesnt implement such function like `scSplit -v` or `scSplit -h`...